### DISCUSSION TITLE
Long Range Forecasts

### DISCUSSION THREAD

**Discussion by wlink99** [weather-gov/api#712] [Q&A] [answered]
Is there a way to use the API for the 90-day forecasts and climatic normals? I'm wanting to use them to do some data analysis on the planting and harvesting of corn. I did see some of the normals, but not nearly enough for my purposes and the Iowa Mesonet doesn't seem to have them either. I hope you can help me find a way to do this if the functionality doesn't exist so I can continue my analysis.

**Comment by sullynole** [comment]
Do you have a link to what you are looking for elsewhere?  Th climate products should be available under the products endpoint.

**Reply by wlink99** [reply]
https://www.weather.gov/wrh/Climate?wfo=fsd and selecting the Daily and Monthly normals option for the normals and https://www.cpc.ncep.noaa.gov/products/predictions/long_range/interactive/index.php for the Long Range Predictions. The CF6 product only covers the ASOS stations with CPC centers. I didn't see one on the API site for the 90-day forecasts.

**Reply by sullynole** [reply, marked as answer]
The CF6 is available in API, but since those links are not text products I am not sure you will find similar data in API.

**Reply by wlink99** [reply]
Thank you so much for your response, I will do my best to find a different way to incorporate that data.

**Reply by mbg206** [reply]
No idea if this helps at all but some data from here might be useful: https://mapservices.weather.noaa.gov/vector/rest/services/outlooks

It has seasonal temp and precip outlooks (higher/lower than normal) which isn't much but is something

**Reply by wlink99** [reply]
Helped quite a bit, I had found it earlier but didn't know I could do anything with it, but geopandas works fine with it. Also found normals from NCEI, but I still need to find something that connects their station codes to ASOS and COOP ones.
